setup.cfg

```
@@ -30,7 +30,7 @@ python_requires = >=3.7,<3.12
 install_requires =
     urllib3~=1.26,>=1.21.1
```

I don't think it's a necessary restriction anymore.

```suggestion
    urllib3>=1.21.1
```